product_docs/docs/bdr/4/conflicts.mdx

```diff
@@ -651,191 +642,189 @@ node.
+For `INSERT`/`INSERT` conflicts, use [global sequences](sequences)
+to prevent this type of conflict.
 
-For applications that assign relationships between objects, e.g. a room
-booking application, applying update_if_newer may not give an acceptable
-business outcome, i.e. it isn't useful to confirm to two people separately
+For applications that assign relationships between objects, such as a room
+booking application, applying `update_if_newer` might not give an acceptable
+business outcome. That is, it isn't useful to confirm to two people separately
 that they have booked the same room. The simplest resolution is to use Eager
-replication to ensure that only one booking succeeds. More complex ways
-might be possible depending upon the application, e.g. assign 100 seats
-to each node and allow those to be booked by a writer on that node, but if
+Replication to ensure that only one booking succeeds. More complex ways
+might be possible depending on the application. For example, you can assign 100 seats
+to each node and allow those to be booked by a writer on that node. But if
 none are available locally, use a distributed locking scheme or Eager
-replication once most seats have been reserved.
+Replication once most seats are reserved.
 
-Another technique for ensuring certain types of update only occur from one
-specific node would be to route different types of transaction through
+Another technique for ensuring certain types of updates occur only from one
+specific node is to route different types of transactions through
 different nodes. For example:
 
--   receiving parcels on one node, but delivering parcels via another node.
--   a service application where orders are input on one node, work is
-    prepared on a second node and then served back to customers on another.
+-   Receiving parcels on one node but delivering parcels using another node
+-   A service application where orders are input on one node, work is
+    prepared on a second node, and then served back to customers on another
 
-The best course of action is frequently to allow conflicts to occur and
+Frequently, the best course is to allow conflicts to occur and
 design the application to work with BDR's conflict resolution
 mechanisms to cope with the conflict.
 
-## Conflict Detection
+## Conflict detection
 
 BDR provides these mechanisms for conflict detection:
 
--   [Origin Conflict Detection](#origin-conflict-detection) (default)
--   [Row Version Conflict Detection](#row-version-conflict-detection)
--   [Column-Level Conflict Detection](column-level-conflicts)
+-   [Origin conflict detection](#origin-conflict-detection) (default)
+-   [Row version conflict detection](#row-version-conflict-detection)
+-   [Column-level conflict detection](column-level-conflicts)
 
-### Origin Conflict Detection
+### Origin conflict detection
 
-(Previously known as Timestamp Conflict Detection, but this was confusing.)
+(Previously known as timestamp conflict detection.)
```

Comment:
I'd just remove this sentence completely, when we are at it (it has been known longer as origin conflict detection than it was known as timestamp conflict detection at this point)